includes/Classifai/Providers/Azure/ComputerVision.php

```diff
@@ -56,9 +56,69 @@ public function can_register() {
 	 */
 	public function register() {
 		add_filter( 'wp_generate_attachment_metadata', [ $this, 'process_image' ], 10, 2 );
+		add_action( 'add_meta_boxes_attachment', [ $this, 'setup_attachment_meta_box' ] );
+		add_action( 'edit_attachment', [ $this, 'maybe_rescan_image' ] );
 		add_filter( 'posts_clauses', [ $this, 'filter_attachment_query_keywords' ], 10, 1 );
 	}
 
+	/**
+	 * Adds a meta box for rescanning options if the settings are configured
+	 */
+	public function setup_attachment_meta_box() {
+		add_meta_box(
+			'attachment_meta_box',
+			__( 'Azure Computer Vision Scan' ),
+			[ $this, 'attachment_data_meta_box' ],
+			'attachment',
+			'side',
+			'high'
+		);
+	}
+
+	/**
+	 * Display meta data
+	 *
+	 * @param \WP_Post $post The post object.
+	 */
+	public function attachment_data_meta_box( $post ) {
+		$captions = get_post_meta( $post->ID, 'classifai_computer_vision_captions', true ) ? __( 'Rescan Captions', 'classifai' ) : __( 'Generate Captions', 'classifai' );
+		$tags     = get_post_meta( $post->ID, 'classifai_computer_vision_image_tags', true ) ? __( 'Rescan Tags', 'classifai' ) : __( 'Generate Tags', 'classifai' );
```

Comment:
Delete alt text of an image > save > still show `Rescan Captions`, same for tags.
The captions and tags still remain in meta after delete, but the button labels are a little bit confused.

Reply:
@dinhtungdu I'm not sure what you mean here :)
Are you suggesting that the text of the buttons should be dependent on if there is existing data or not?

Reply:
@ryanwelcher I mean when the alt text is deleted, the label should say Generate instead of Rescan. Note that when we delete the alt text of an image, the `classifai_computer_vision_captions` remains unchanged. This is a minor issue btw.

Reply:
I've updated the logic accordingly. Thanks for the great suggestion!